programs/marinade-finance/src/instructions/crank/deactivate_stake.rs

```diff
@@ -59,8 +58,12 @@ pub struct DeactivateStake<'info> {
         bump = state.stake_system.stake_deposit_bump_seed
     )]
     pub stake_deposit_authority: UncheckedAccount<'info>,
-    #[account(mut)]
-    pub split_stake_account: Signer<'info>,
+    #[account(
```

Comment:
Can you check if `init` is applicable here? I think it should be used only for accounts owned by the program.
if split_stake_account is sent with 0 lamports, `init` macro will create the account and **assign it to marinade-program-id** .
That's not correct, we need split_stake_account to be owned by the stake-program